### PR TITLE
Another attempt to fix circle.

### PR DESCRIPTION
**Pull request by jaeilepp**
Perhaps this'll work...

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -37,9 +37,8 @@ dependencies:
       else
         echo "Conda already set up.";
       fi
-    - conda install -n circleenv --yes numpy scipy scikit-learn matplotlib sphinx=1.4.1 pillow six IPython pandas;
-    - conda install -n circleenv --yes traits=4.6.0 traitsui=5.1.0;
-    - conda install -n circleenv --yes mayavi;
+    - conda install -n circleenv --yes numpy scipy scikit-learn matplotlib sphinx=1.4.1 pillow six IPython pandas apptools=4.4.0 envisage=4.5.1 pyface=5.1.0 traits=4.6.0 traitsui=5.1.0 vtk=5.10.1;
+    - pip install mayavi==4.4.4;
```
Comment (Eric89GXL) [inline circle.yml]: Can we install `mayavi` with conda but then `pip --upgrade` whatever package it gets errors on?

**Comment by Eric89GXL** [comment]
BTW the way to properly test this is probably to tell Circle to run it without using cache, which you can do on the web interface

**Comment by Eric89GXL** [comment]
I guess we should do the same thing for `master`. It's possible that one could work already if we just run without using the existing cache. I'll try it.

**Comment by Eric89GXL** [comment]
It looks like `master` was working, but unfortunately there is a bug to fix

https://circleci.com/gh/mne-tools/mne-python/3339

But it looks like these changes could be worthwhile anyway. @jaeilepp you on this new bug?

**Comment by jaeilepp** [comment]
Fixed.

**Comment by agramfort** [comment]
CIs are happy

+1 for merge

File: mne/viz/misc.py [hunk 2]
```diff
@@ -110,6 +110,8 @@ def plot_cov(cov, info, exclude=[], colorbar=True, proj=False, show_svd=True,
 
     fig_cov, axes = plt.subplots(1, len(idx_names),
                                  figsize=(2.5 * len(idx_names), 2.7))
+    if isinstance(axes, plt.Axes):
+        axes = [axes]
```
Comment (Eric89GXL) [inline mne/viz/misc.py]: a clearer way is to do `axes = plt.subplots(..., squeeze=False)[0]`, then you are guaranteed it is a 1D array with 1 or more elements.

File: circle.yml (hunk 1 above)
Comment (Eric89GXL) [inline circle.yml]: Still curious about this, too

**Comment by Eric89GXL** [comment]
Taking over in #3868 so that we can fix Circle ASAP

**Pull request closed without merging**